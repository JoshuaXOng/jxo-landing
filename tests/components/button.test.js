"use strict"; 

import React from "react";
import renderer from 'react-test-renderer'
import Button from "../../src/components/button/button";

test.skip('Button text displays as per args', () => {
    
    const component = renderer.create(
        <Button text={"GLOOOBER"}></Button>    
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});

test.skip("Button recieves correct styling", () => {

    const component = renderer.create(
        <Button size={{sd: 21321}} 
            bColor={"grey"}
        >
        </Button>    
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

})