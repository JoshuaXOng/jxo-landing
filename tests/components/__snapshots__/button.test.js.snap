// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button recieves correct styling 1`] = `
<button
  className="jxo-button jxo__button"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "grey",
      "height": undefined,
      "width": undefined,
    }
  }
/>
`;

exports[`Button text displays as per args 1`] = `
<button
  className="jxo-button jxo__button"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": null,
      "height": null,
      "width": null,
    }
  }
>
  GLOOOBER
</button>
`;
